pkg/kubevip/config_environment.go

```diff
@@ -316,11 +319,17 @@ func ParseEnvironment(c *Config) error {
 	// Routing Table ID
 	env = os.Getenv(vipRoutingTableID)
 	if env != "" {
-		i, err := strconv.ParseInt(env, 10, 32)
+		i, err := strconv.ParseInt(env, 10, 64)
 		if err != nil {
 			return err
 		}
-		c.RoutingTableID = int(i)
+		if i >= 0 && i <= math.MaxInt {
+			c.RoutingTableID = int(i)
+			return nil
+		} else if i < 0 {
+			return fmt.Errorf("no support of negative [%d] in env var %q", i, vipRoutingTableID)
+		}
+		return fmt.Errorf("no support for int64 natively on system only [int%d]", bits.OnesCount(math.MaxUint))
```

Comment:
missing a comma here, and for consistency I would use `math.MaxInt` here as well, if that yields the same result.

Reply:
nope as it has a signing bit which is zero for a positive number.

Reply:
fixed the error message.